### DISCUSSION TITLE
Testing via postman

### DISCUSSION THREAD

**Discussion by officialgabzz** [frankie567/starlette-csrf#15] [Q&A] [answered]
Hello, 
thanks for this awesome library. 
I have been trying to test it with Postman which has been a blocker. I set the header x-csrftoken but always get CSRF token verification failed.
![image](https://github.com/frankie567/starlette-csrf/assets/29066025/1bead240-251c-4bf1-bde4-3575efec1cf1)
 Appreciate response in advance

**Comment by frankie567** [comment, marked as answer]
Hi! Make sure you send *both* the header and the cookie :)

**Comment by officialgabzz** [comment]
Awesome, Thank you. works fine. 
![image](https://github.com/frankie567/starlette-csrf/assets/29066025/c4ac28a3-5b5c-4523-8cbd-8b3f6c2adaf1)
putting it here incase anyone else needs it.
